src/ansible.coffee

```diff
@@ -35,3 +35,5 @@ module.exports = (robot) ->
     command = msg.match[1]
 
     runAnsiblePlaybook msg, command
+
+  robot.on 'ansible:runPlaybook', runAnsiblePlaybook
```

Comment:
Yeah exactly, I figured that signature was fine.  I'm using it successfully here: https://github.com/teamgantt/henry/pull/1/commits/b549cb0b0e957514046c19c39b1daa232a6b18b3#diff-557f2b0e779e7ceb19359fdea7e811abR17